pkg/resources/data.go

```diff
@@ -785,6 +785,15 @@ func (data *TemplateData) GetEnvVars() ([]corev1.EnvVar, error) {
 	}
 	if cluster.Spec.Cloud.Kubevirt != nil {
 		vars = append(vars, corev1.EnvVar{Name: "KUBEVIRT_KUBECONFIG", ValueFrom: refTo(KubeVirtKubeconfig)})
+
+		if dc.Spec.Kubevirt != nil {
+			allowPVCClone := dc.Spec.Kubevirt.Images.HTTP != nil && dc.Spec.Kubevirt.Images.HTTP.ImageCloning.Enabled
+			vars = append(vars, corev1.EnvVar{Name: "KUBEVIRT_ALLOW_PVC_CLONE", Value: strconv.FormatBool(allowPVCClone)})
```

Comment:
Will need to update with: `KUBEVIRT_ENABLE_PVC_CLONE` and `KUBEVIRT_PVC_CLONE_NAMESPACES` when machine-controller PR merged and bumped in this PR (or the next one ? )